### PR TITLE
Ammo Crafting Fixes and Tweaks

### PR DESCRIPTION
**Pull request by SwiGs1**
## Description
These are the following changes to ammo crafting: To craft any type of ammunition, you will always need both that ammo type's equivalent 'Guns and Ammo' book, as well as gunpowder or blackpowder which can only be acquired by using a wrench on bullets to retrieve (or with a chem dispenser) and metal, all of these in very large quantities. This change reverts ammo to their BD recipes, which required only the necessary amount of metal, screwdriver and the appropriate ammo bench. No longer is black powder required either. It negates the mandatory needs that you have someone capable of using the chem dispenser in your faction and the tedious task of having to wrench individual bullets of the ground isn't as appealing and is not used and if so it is very rare at least from what I've seen and I have no idea why black powder is being used for ballistic weapons that should use smokeless powder used in today's gun cartridges instead of messy, dusty black powder. Also having Guns and Ammo be a requirement for crafting ammo is something I don't think works, as you need all four of them to craft every type of ammo, and there simply aren't enough book spawns on the maps. Also from what I've seen between the NCR and Legion meta, being much faster than I'm personally used to with both factions often clashing and wiping each other off around the 30-40 minutes mark, and it becomes far too much time-costly preparation and chance to properly produce ammunition. Also, Ammo Benches are actually required to craft ammo boxes, while workbenches will be used to craft the ammo storage units. For some reason the current reloading bench system does nothing It has no use, and is only referenced twice from what I can find: one in code/_defines/tools.dm and one in code/modules/crafting/guncrafting.dm. Seems to be never used outside of a define. Anyway, that patches that. 

## Motivation and Context
Read the description. Pretty much sums it up. As it's a shitty DR system that is smooth brain. 

## How Has This Been Tested?
In the Swigs Industries Laboratory on a local instance.

## Screenshots (if appropriate):
Screenshot (731)
Screenshot (734)
Screenshot (730)

## Changelog (necessary)
:cl:
tweak: metal, screwdriver and the appropriate workbench or ammo bench is now used for crafting no more need for Guns and Ammo books 
fix: the appropriate ammo benches are required to craft the appropriate types of ammo. 
tweak: Blackpowder not needed as a requirement to craft ammo still can be used to craft bombs and stuff 
/:cl:

### CODE REVIEW

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -1836,6 +1809,15 @@
...
 
+	/datum/crafting_recipe/antivenom
+	name = "Antivenom"
+	result = /obj/item/reagent_containers/glass/bottle/antivenom
+	reqs = list(/obj/item/stack/sheet/glass = 1,
+				/obj/item/reagent_containers/food/snacks/grown/feracactus = 2,
+				/obj/item/reagent_containers/food/snacks/grown/fungus = 1 )
+	time = 55
+	category = CAT_MEDICAL
+
```
Comment (Afya212) [inline code/modules/crafting/recipes.dm:1820]: Why is this in an ammo change PR? Reeee

Comment (SwiGs1) [reply code/modules/crafting/recipes.dm:1820]: the fuck?

Comment (SwiGs1) [reply code/modules/crafting/recipes.dm:1820]: no fucking idea.

Comment (SwiGs1) [reply code/modules/crafting/recipes.dm]: Fixed now i guess

**Pull request merged**